personalize: docs new addition (#47)
rudderlabs/rudderstack-docs: destinations/aws-personalize.md

  * **Information on Dataset Group**

    * **TrackingId**: Enter the tracking ID that you generated in the first step 
    * **Type Of Event**: Enter the name of event you want to send for this destination.
   
     {% hint style="info" %}

>      As event name is a mandatory field here, that means you can only send `track` events. Also remember that, inside the payload you have to specify the **same** event name, which you have used while configuring the destination in [RudderStack dashboard](https://app.rudderlabs.com/)

Reviewer: ```suggestion
     As event name is a mandatory field here, you can only send `track` events. Also, you need to specify the **same** event name, which you have used while configuring the destination on [RudderStack dashboard](https://app.rudderstack.com/)
```
